packages/pages/src/dev/server/server.ts

```diff
@@ -43,20 +48,24 @@ export const createServer = async (
   // register vite's middleware
   app.use(vite.middlewares);
 
-  // Ignore favicon requests if it doesn't exist
+  // Load favicon for index page
+  app.use("/favicon.png", (req, res) => {
+    const faviconRelativePath = Favicon.default.slice(3); // the import has one too many "../"
```

Comment:
This seems overly complicated to serve back a file. Is there a different way that works without need to change the paths? I know the favicon.ico is already being requested, hence the ignore part below. If our file is a .ico instead of .png would it just work?